src/test/ui/issues/issue-67037-pat-tup-scrut-ty-diff-less-fields.stderr

```diff
@@ -17,6 +17,15 @@ LL | struct P<T>(T); // 1 type parameter wanted
 ...
 LL |     let P() = U {};
    |         ^^^ expected 1 field, found 0
+   |
+help: use `_` to explicitly ignore each field
+   |
+LL |     let P(_) = U {};
+   |           ^
+help: use `..` to ignore all fields
+   |
+LL |     let P(..) = U {};
+   |           ^^
```

Comment:
I think that we'll need to special case when there's only one field missing (and all the other preceding fields aren't `_`) and _only suggest `_`. To clarify, I think that in _this_ case we should _only_ provide the first suggestion, not the `..` one. That being said, it is something I can look at myself if you're itching to merge asap.

It also feels like we could deal with language for a single field. My ideal output for _this_ case would be:

```
help: use `_` to explicitly ignore the missing field
   |
LL |     let P(_) = U {};
   |           ^
```

Reply:
> To clarify, I think that in this case we should only provide the first suggestion, not the .. one.

I actually think we _should_ suggest `..` in this case. It's very possible that someone didn't list any fields because they want to match on the outer structure, and so they just want to ignore all of them. We should definitely suggest `..` in that case.

And also, as you said, I would like to get this merged soon. I appreciate the desire to make diagnostics as good as possible - I would like to as well - but I feel that further changes would have diminishing returns and increase the risk of introducing bugs. Also, this change has taken way longer than I anticipated and I would like to spend my time on some other things :)